core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -22,55 +25,62 @@ import scala.util.Try
  */
 class HttpExecutionServiceDAO( executionServiceURL: String, submissionTimeout: FiniteDuration )( implicit val system: ActorSystem ) extends ExecutionServiceDAO with DsdeHttpDAO with Retry with FutureSupport with LazyLogging {
 
+  //sendReceive, but with gzip.
+  //Two versions, one of which overrides the timeout.
```

Comment:
A few comments about implicits:
1. I think the `implicit refFactory: ActorRefFactory` parameter is unnecessary since the `ActorSystem` is already in scope at the class level.
2. It looks like every method here does `import system.dispatcher` to obtain an ExecutionContext. That could probably be put at the class level instead. 
3. Same with the default timeout

Then I think you could write something like:
```
class HttpExecutionServiceDAO(...)( implicit val system: ActorSystem ) extends ... {
  import system.dispatcher
  implicit val timeout: Timeout = 60 seconds

  def gzSendReceive(implicit timeout: Timeout) =
    addHeaders(`Accept-Encoding`(gzip)) ~> sendReceive ~> decode(Gzip)

  ...

  // Now you can still write gzSendReceive and gzSendReceive(someOtherTimeout)
}
```